src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs

```diff
@@ -105,6 +106,12 @@ public class JsonInputFormatter : TextInputFormatter
         }
 
         /// <summary>
+        /// Gets the flag indicating if it honors the value in setting <see cref="MvcOptions.ConvertAllExceptionsOfInbuiltInputFormattersToModelStateErrors"/>.
+        /// Default value is <see langword="true"/>.
```

Comment:
Should you say `true if the current instance is the exact type `JsonInputFormatter` and not a derived type`?

Reply:
Check my doc comments on IFormatterExceptionPolicy and let me know if its not ok